worlds/AutoWorld.py

```diff
@@ -4,12 +4,11 @@
 import logging
 import pathlib
 import random
-import re
 import sys
 import time
 from dataclasses import make_dataclass
-from typing import (Any, Callable, ClassVar, Dict, FrozenSet, List, Mapping,
-                    Optional, Set, TextIO, Tuple, TYPE_CHECKING, Type, Union)
+from typing import Any, Callable, ClassVar, Dict, FrozenSet, List, Mapping, Optional, Set, TextIO, Tuple, \
+                   TYPE_CHECKING, Type, Union
```

Comment:
PEP8 says this:
> The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

Reply:
(The parentheses way is also my personal preference.)

Reply:
Fair enough, my personal preference shouldn't override PEP8 I suppose for something silly like this.